src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Resources.xaml

```diff
@@ -213,6 +213,7 @@
               Grid.Column="1"
               Focusable="False"
               Margin="{TemplateBinding Padding}"
+              RecognizesAccessKey="True"
```

Comment:
👀

Reply:
Added to support access keys in Checkbox, disabled in this style before this change.